### PR TITLE
 refactor: Material animations now use handles directly

### PR DESCRIPTION
**Pull request by Rhuagh**
This change is [Reviewable](https://reviewable.io/reviews/amethyst/amethyst/1089)

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -18,8 +18,12 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ### Changed
 
+* `SpriteRenderPrimitive::SpriteSheet` now takes `Handle<SpriteSheet>` instead of a `u64` ID.
+
 ### Removed
 
+* `SpriteSheetSet` is removed as it is no longer needed.
+
 ### Fixed
```
Comment (jojolepro) [inline docs/CHANGELOG.md]: Will need a link to the pr :)

**Comment by LucioFranco** [comment]
@Rhuagh fix the conflicts and lets merge!

**Comment by azriel91** [comment]
https://github.com/Rhuagh/amethyst/pull/7: in case you come here first.

**Pull request merged**